.github/workflows/github-page.yml

```diff
@@ -82,6 +57,22 @@ jobs:
         sudo apt-get -y remove --purge software-properties-common
         sudo apt-get -y autoremove
 
+    - name: Install secp256k1
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
Just noted that this was `ubuntu-20.04` before, which is incorrect in this PR. Now fixed.